core/base/inc/TString.h

```diff
@@ -44,21 +45,26 @@ TString operator+(const TString &s1, const TString &s2);
 TString operator+(const TString &s,  const char *cs);
 TString operator+(const char *cs, const TString &s);
 TString operator+(const TString &s, char c);
-TString operator+(const TString &s, Long_t i);
-TString operator+(const TString &s, ULong_t i);
-TString operator+(const TString &s, Long64_t i);
-TString operator+(const TString &s, ULong64_t i);
 TString operator+(char c, const TString &s);
-TString operator+(Long_t i, const TString &s);
-TString operator+(ULong_t i, const TString &s);
-TString operator+(Long64_t i, const TString &s);
-TString operator+(ULong64_t i, const TString &s);
 Bool_t  operator==(const TString &s1, const TString &s2);
 Bool_t  operator==(const TString &s1, const char *s2);
 Bool_t  operator==(const TSubString &s1, const TSubString &s2);
 Bool_t  operator==(const TSubString &s1, const TString &s2);
 Bool_t  operator==(const TSubString &s1, const char *s2);
+/*
```

Comment:
Should we remove this commented out code or is it a reminder for future/alternative implementation?